glambie/data/data_catalogue.py

```diff
@@ -53,19 +53,39 @@ def from_dict(meta_data_dict: dict) -> DataCatalogue:
         DataCatalogue
             data catalogue containing the metadata of datasets, the actual timeseries data will lazy loaded
         """
-        basepath = os.path.join(*meta_data_dict['basepath'])
+        base_path = os.path.join(*meta_data_dict['basepath'])
```

Comment:
This is a good change, can we change it in meta_data_dict as well?

Reply:
Yes, done